bundles/org.openhab.core.rulestest/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Can you please use two spaces for indentation?

We can fix this later by using an automatic task (would like to do it for the other ones, too).

Reply:
ok